processmanager/manager.go

```diff
@@ -286,6 +286,17 @@ func (pm *ProcessManager) ConvertTrace(trace *host.Trace) (newTrace *libpf.Trace
 
 			newTrace.AppendFrameFull(frame.Type, fileID,
 				relativeRIP, mappingStart, mappingEnd, fileOffset)
+		case libpf.CUDA:
+			name := trace.CustomLabels["_temp_cuda_kernel"]
```

Comment:
Should this be made a field on `trace` rather than being stuffed in `CustomLabels`? What do you think? We already have interpreter-specific stuff in there, e.g. `APMTransactionID`.